lib/AppInfo/Application.php

```diff
@@ -60,6 +64,12 @@ public function register(IRegistrationContext $context): void {
 		/** @psalm-suppress TooManyArguments */
 		$context->registerMiddleware(PerformanceMonitoringMiddleware::class, true);
 		$context->registerEventListener(AddContentSecurityPolicyEvent::class, CustomCspListener::class);
+		if (class_exists(BeforeJobExecutedEvent::class)) {
```

Comment:
Should not be needed, the class name can be used without existance, just psalm complains

Reply:
I did it for Psalm but it complains in any case